src/main/java/uk/gov/pay/ledger/payout/state/PayoutState.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.pay.ledger.payout.state;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.pay.ledger.event.model.SalientEventType;
+import java.util.Map;
+
+import static java.util.Arrays.stream;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
+public enum PayoutState {
+    UNDEFINED("undefined", false),
+    IN_TRANSIT("intransit", false),
+    PAID_OUT("paidout", true),
+    FAILED("failed", true);
+
+    private final String status;
+    private final boolean finished;
```

Comment:
is there any significance of `finished` for payouts ?

Reply:
yeah there are still terminal (completed) and in progress states there just happens to only be one in progress state at the moment! 

Having this in from the beginning will still allow the code downstream to say `if in progress` rather than matching to a specific state.

If we think this we can simplify this further later we can just serialise the JSON to the "status" and remove this param.

Reply:
Makes sense. It's is possible that payout is created but won't be finished for a few hours. Looks useful to have especially if we this available via API